### DISCUSSION TITLE
Colab crashing while using PaddleOCR

### DISCUSSION THREAD

**Discussion by clintonvanry01** [PaddlePaddle/PaddleOCR#8181] [General]
Hi,

Is anyone else experiencing this with google Colab?
It is also reported on  stack overflow

**Comment by 708kelly** [comment]
maybe you can try make the batch num smaller

**Reply by clintonvanry01** [reply]
Hi,

there is no training involved and it is just the basics of paddleocr
here is a link that you can use
https://colab.research.google.com/drive/1Xsjlbd3Kiqwu-_h-Y--gSCLHMSh0yg2w?usp=sharing

and you will notice it crashes

**Reply by OttomanZ** [reply]
@clintonvanry01 Seems fine to me, I think that you might be running out of RAM or GPU memory. In both cases, Google Colab restarts and shows you a message about whether you would like to upgrade. If that's not the case then this issue would require further investigation. 

*On my end with an RTX 3060* it worked just fine.
